index.js

```diff
@@ -24,12 +24,12 @@ const print = (query) => console.log(query);
 	}
   const commentHTML = await commentFetch.text();
   const dom = new JSDOM(commentHTML);
-  const items = dom.window.document.getElementsByClassName("top-level-reply")
+  const items = dom.window.document.getElementsByClassName("top-level-reply");
   for(let elID in items) {
     const element = items[elID];
     if (typeof element == "function") break;
     const commentPoster = element.getElementsByClassName("comment")[0].getElementsByTagName("a")[0].getAttribute('data-comment-user')
-    print(`Current Comment Poster: ${commentPoster}`)
   };
+  print(`Found ${items.length} comments!`)
```

Comment:
Use `console.log` or define a passthrough function globally.

Reply:
What do you mean?

Reply:
I'm replying this through VS Code. So easy to code at the same time :smile:

Reply:
> I'm replying this through VS Code. So easy to code at the same time 😄

How?

Reply:
Github extension

Reply:
> Github extension

Oh! Do you use the live share extension?

Reply:
I have it, but I use it very rarely.

Reply:
> I have it, but I use it very rarely.

What's it like? If it works on the [web version](https://vscode.dev/) I might use it to sync directories between computers.

Reply:
> Github extension

I like it! Thanks for the tip.